e2e/pages/card-page.ts

```diff
@@ -35,6 +34,13 @@ export class CardPage {
     );
     await expect(this.votes).toHaveCount(states.length);
   }
+
+  async kickUser(name: string) {
+    const kickButton = this.votes
+      .filter({ has: this.page.locator(`:has-text("${name}")`) })
+      .locator('td:nth-child(3) button');
```

Comment:
The button has the title attribute `Kick ${name}`. Can we find the button by this title instead?

Reply:
Great idea, done!